capa/ida/plugin/view.py

```diff
@@ -198,14 +192,14 @@ def load_preview_meta(self, ea, author, scope):
             "    name: <insert_name>",
             "    namespace: <insert_namespace>",
             "    authors:",
-            "      - %s" % author,
-            "    scope: %s" % scope,
+            f"      - {author}",
+            f"    scope: {scope}",
             "    references:",
             "      - <insert_references>",
             "    examples:",
-            "      - %s:0x%X" % (capa.ida.helpers.get_file_md5().upper(), ea)
+            f"      - {capa.ida.helpers.get_file_md5().upper()}:{hex(ea)}"
             if ea
-            else "      - %s" % (capa.ida.helpers.get_file_md5().upper()),
+            else f"      - {capa.ida.helpers.get_file_md5().upper()}",
             "  features:",
```

Comment:
i dont like mixing f-strings and literal strings like this since it breaks up the horizontal alignment of the strings. would you please revert this one?